test/cases/testdata/array/test-array-0052.yaml

```diff
@@ -0,0 +1,52 @@
+cases:
+- note: 'array/reverse_123'
+  query: data.generated.p = x
```

Comment:
I'd name this something else than `generated`

Reply:
Changed to `test`